src/client/ServiceBusClient.ts

```diff
@@ -200,15 +200,36 @@ export default class ServiceBusClient implements IServiceBusClient {
                 body: body,
                 contentType: contentType
             });
-            
+
 
             await client.close();
-           
+
         } catch{
 
             if (client) {
                 await client.close();
             }
         }
     }
+
+    public async deleteMessage(topic: string, subscription: string, messageId: string): Promise<void> {
+        const client = SBC.ServiceBusClient.createFromConnectionString(this.connectionString);
+        const subscriptionClient = client.createSubscriptionClient(topic, subscription);
+
+        let receiver = subscriptionClient.createReceiver(SBC.ReceiveMode.peekLock);
+        try {
+            const messages = await receiver.receiveMessages(1000, 10);
```

Comment:
I had issues with this API before. Are you sure there is no other way around this , but retrieving all the messages?
We need to make sure that the REST API does not have a valid verb for this.